go.mod

```diff
@@ -0,0 +1,109 @@
+module github.com/silinternational/scheduled-codeship-build
+
+go 1.17
+
+require (
+	github.com/aws/aws-lambda-go v1.28.0
+	github.com/codeship/codeship-go v0.0.0-20201103162013-82059492d39f
+)
+
+require (
```

Comment:
Wow... that's a disconcerting number of dependencies for how simple the API calls are to do manually. Not a show-stopper, but slightly surprising.

Reply:
That's not a criticism of your or your work @Baggerone. I know they're sub-dependencies, not direct.